src/ChildProcessUtilities.js

```diff
@@ -4,6 +4,14 @@ export default class ChildProcessUtilities {
   static exec(command, opts, callback) {
     return child.exec(command, opts, (err, stdout, stderr) => {
       if (err != null) {
+
+        // If the error from `child.exec` is just that the child process
+        // emitted too much on stderr, then that stderr output is likely to
+        // be useful.
+        if (/^stderr maxBuffer exceeded/.test(err.message)) {
+          err = `Error: ${err.message}.  Partial output follows:\n\n${stderr}`;
+        }
+
         callback(err || stderr);
```

Comment:
Hey, I don't want to let this slip through the cracks, but I'm actually not sure what the TODO is here. 😅